src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMappingTest.java

```diff
@@ -186,14 +185,22 @@ void buildHearingPartiesDetailsOtherParties() {
     }
 
     @DisplayName("When a valid hearing wrapper with joint party given buildHearingPartiesDetails returns the correct Hearing Parties Details")
-    @ParameterizedTest
-    @EnumSource(value = YesNo.class)
```

Comment:
You need to cover all conditions in your tests:
https://sonarcloud.io/component_measures?metric=new_coverage&selected=SSCSHA%3Asrc%2Fmain%2Fjava%2Fuk%2Fgov%2Fhmcts%2Freform%2Fsscs%2Fhelper%2Fmapping%2FHearingsPartiesMapping.java&view=list&pullRequest=127&id=SSCSHA